docs/semgrep-app/scanning-cloud-repositories.md

```diff
@@ -0,0 +1,26 @@
+---
+slug: scanning-cloud-repositories 
```

Comment:
How does one get to this page? I couldn’t find a link to it.

Reply:
My mistake, I was thinking of cutting up "Getting started" into two separate workflows. 🤔 I don't know how I feel about the length (affects the perception of time it takes to complete) vs completeness.

Reply:
I have deleted this. Resolving.